src/View/Exit/_form.html.twig

```diff
@@ -9,24 +9,24 @@
 		<form class="add_form" method="post" enctype="multipart/form-data">
 			<div class="text-center">
 				<div class="mb-4">
-					<label class="col-6 col-lg-6"  for="name"><strong>Nom</strong>
+					<label class="col-6 col-lg-6"  for="name"><strong>Nom *</strong>
 					</label>
 					<input class="col-11 col-lg-8" type="text" value="{{ exit.name }}" name="name" id="name" maxlength="150" placeholder="Juste un saut" required>
 					</div>
 				<div class="mb-4">
-					<label class="col-6 col-lg-1 me-lg-1 text-lg-end" for="department"><strong>Département</strong>
+					<label class="col-6 col-lg-1 me-lg-1 text-lg-end" for="department"><strong>Département*</strong>
```

Comment:
you miss one space between "Département" and "*"

Reply:
Fixed chef !